### PR TITLE
fix blank project change #3106

### PR DESCRIPTION
**Pull request by btyanagawamg**
Please review #3106 .

### CODE REVIEW

File: source/ArchitectureInDetail/WebApplicationDetail/TilesLayout.rst [hunk 1]
```diff
@@ -329,19 +329,13 @@ Tilesのカスタムタグの詳細は、\ `こちら <http://tiles.apache.org/f
...
-  <!--<![endif]-->
   <head>
   <meta charset="utf-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
   <meta name="viewport" content="width=device-width" />
-  <link rel="stylesheet"
-      href="${pageContext.request.contextPath}/resources/app/css/styles.css"
-      type="text/css" media="screen, projection">
+  <link rel="stylesheet" href="${pageContext.request.contextPath}/resources/app/css/styles.css"
+      media="screen, projection">
```
Comment (yoshikawaa) [inline source/ArchitectureInDetail/WebApplicationDetail/TilesLayout.rst]: - [ ] See https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/4b67c6939ced583741f5d4c2917a66d9ac3c1e75/projectName-web/src/main/webapp/WEB-INF/views/layout/template.jsp#L14

File: source/ImplementationAtEachLayer/CreateWebApplicationProject.rst [hunk 2]
```diff
@@ -450,8 +450,7 @@ Maven Archetypeで作成したプロジェクトでは、エラーの種類毎
     <head>
     <meta charset="utf-8">
     <title>Resource Not Found Error!</title>
-    <link rel="stylesheet"
-        href="${pageContext.request.contextPath}/resources/app/css/styles.css">
+    <link rel="stylesheet" href="${pageContext.request.contextPath}/resources/app/css/styles.css">
     </head>
     <body>
         <div id="wrapper">
```
Comment (yoshikawaa) [inline source/ImplementationAtEachLayer/CreateWebApplicationProject.rst]: - [ ] See https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/4b67c6939ced583741f5d4c2917a66d9ac3c1e75/projectName-web/src/main/webapp/WEB-INF/views/common/error/resourceNotFoundError.jsp#L16-L39

File: source/ArchitectureInDetail/WebApplicationDetail/TilesLayout.rst (hunk 1 above)
Comment (btyanagawamg) [reply source/ArchitectureInDetail/WebApplicationDetail/TilesLayout.rst]: https://github.com/terasolunaorg/guideline/pull/3110/files#diff-4d3e92ad1ab7cd69b5f3abc644d198c3R344
fixed

File: source/ImplementationAtEachLayer/CreateWebApplicationProject.rst (hunk 2 above)
Comment (btyanagawamg) [reply source/ImplementationAtEachLayer/CreateWebApplicationProject.rst]: https://github.com/terasolunaorg/guideline/pull/3110/files#diff-42ac2a1ab63cfda6f129558edb557b3cR463
fixed

**Comment by btyanagawamg** [comment]
@yoshikawaa 
I fixed to all indications .
Please merge .

**Pull request merged**